test/quantization/core/test_quantized_op.py

```diff
@@ -3661,6 +3661,93 @@ def test_embedding_bag_2d_indices(self):
         torch.testing.assert_close(result, qresult, atol=0.05, rtol=1e-3)
 
 
+class TestDynamicQuantizedConv(TestCase):
+    def _test_qconv_op_impl(self, q_mod, dq_op, dim, q_engine, dtype):
+        # The goal here is to show that the dynamic op is the same as
+        # calc params->quantize_input->quantized op->dequantize output
+        if q_engine not in torch.backends.quantized.supported_engines:
+            return
+        torch.backends.quantized.engine = q_engine
+        reduce_range = True
+
+        X_fp32 = torch.randn(*([2] * dim))
+        s, z = _calculate_dynamic_qparams(X_fp32, dtype, reduce_range)
+
+        quantized_module = q_mod(2, 3, 1)
+        packed_params = quantized_module._packed_params
+
+        quantized_module.scale, quantized_module.zero_point = s, z
+
+        X_q = torch.quantize_per_tensor(X_fp32, s, z, dtype)
+        Y_q_ref = quantized_module(X_q)
+        Y_ref = torch.dequantize(Y_q_ref)
+
+        X_dq = torch.dequantize(X_q)
+        Y = dq_op(X_dq, packed_params, reduce_range)
+
+        self.assertEqual(Y, Y_ref)
+
+    def test_dynamic_conv1d(self):
+        q_mod = torch.nn.quantized.Conv1d
+        dq_op = torch.ops.quantized.conv1d_dynamic
+        dim = 3
+        dtype = torch.quint8
+
+        for q_engine in ["fbgemm", "qnnpack"]:
+            for i in range(10):
```

Comment:
Why run 10 times? Just with different rand inputs? If so I dont suppose we need to do that.

Reply:
ok, removed